src/demo.scss

```diff
@@ -14,3 +14,15 @@ $fa-font-path : '../node_modules/font-awesome/fonts';
 @import "app/dropdowns/dropdowns.component";
 @import "app/branding/branding.component";
 @import "app/list/list.component";
+
+//global styles for the demo app
+
+.feature-alert.feat-alert-for-inline-element{
```

Comment:
can you add these to the existing `feature-alerts.component.scss` in the app folder